graylog2-web-interface/src/theme/GlobalThemeStyles.jsx

```diff
@@ -1900,7 +1881,7 @@ const GlobalThemeStyles = createGlobalStyle`
     padding: 0;
   }
 
-  .indexer-failures-body select.dynatable-per-page-select {
```

Comment:
should remove entire selector, not just change what it selected.

Reply:
✅